modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/upgrade/registry/DLWebUpgradeStepRegistrator.java

```diff
@@ -49,6 +50,8 @@ public void register(Registry registry) {
 				DLPortletKeys.DOCUMENT_LIBRARY_ADMIN));
 
 		registry.register("1.0.1", "1.0.2", new UpgradePortletPreferences());
+
+		registry.register("1.0.2", "1.1.0", new DLUpgradeProcess());
```

Comment:
The upgrade process focus on the DLFolder, I think it is better to change the DLFolderUpgradeProcess. Moreover,this file should in documet-library-service, not in document-library-web

Reply:
I've updated it.